src/Symfony/Component/Console/Command/Command.php

```diff
@@ -356,6 +356,14 @@ public function complete(CompletionInput $input, CompletionSuggestions $suggesti
         }
     }
 
+    /**
+     * Get the InvokableCommand instance if it exists.
+     */
+    public function getCode()
+    {
+        return $this->code;
+    }
```

Comment:
We can skip the internal `InvokableCommand` and return the callable that has been set.

```suggestion
    public function getCode(): ?callable
    {
        return $this->code?->getCallable();
    }
```